data/pools/transactionPool_test.go

```diff
@@ -40,6 +44,12 @@ func keypair() *crypto.SignatureSecrets {
 	return s
 }
 
+type TestingT interface {
```

Comment:
Is this used anywhere for anything?

Reply:
It allows passing in either a `testing.T` or a `testing.B`.  I think in this PR, only the benchmark code calls it; it's a leftover from the larger PR #150 that also uses this code from tests.